**Jason Junjie Zhu**

[TODO: Rewrite and keep it as concise] I start most of my journeys out of *curiosity* of patterns and connections, and what keeps me focused on a path is how I can graduately build **influence** with a team that is impact-driven. 
I have had the fortune to work with world-class scientists and top-tier product teams. My main contributions are in AI methodology — including network science, unsupervised learning, and large-scale testing — as well as practical AI-driven applications in areas like search engines and biotechnology. 

**Education**  

- **Stanford University**  
  - Ph.D. in Electrical Engineering
  - M.S. in Statistics

- **Olin College of Engineering**  
  - B.S. in Electrical and Computer Engineering  

**Professional Experience**

- **Head of AI/ML**  
  *Nexa AI | Feb 2025 – Present | Cupertino, CA*  
  - Leading a lean and fast-paced team to accelerate Gen-AI edge inferenceon any device.
  - Developing privacy-driven RAGs with efficient AI models with LLMs and on-device vision capabilities.
  - Researching and building agentic applications with new AI protocols (MCP, A2A, etc.)

- **Machine Learning Engineer**  
  *Apple | Jan 2020 – Feb 2025 | Cupertino, CA*  
  - Developed both generative and retrieval-based methods to evaluate ranking and query-understanding ML systems.
  - Analyzed complex production data sets with high-dimensional statistics and large-scale computing.
  - Designed new testing strategies for software services at the intersection of user experience and computational algorithms.

- **Deep Learning Scientist (Internship)**  
  *Illumina | Jun 2017 – Aug 2017 | San Francisco Bay Area*  
  - Wrote customized deep learning software in TensorFlow and experimentation infrastructure for base-calling applications.
  - Systematically combined convolutional neural networks, recurrent neural networks, and residual network architectures to significantly improve accuracy.

- **Data Scientist (Internship)**  
  *10X Genomics | Jun 2016 – Aug 2016 | Pleasanton, CA*  
  - Productionized R software packages for exploratory single-cell RNA sequencing analysis and improved internal Python software pipelines. 

---

**Research Experience**  

- *Department of Statistics, Stanford University | Sep 2016 – Feb 2020* 
  - Developed graph visualizations to interpret and analyze the Gene Ontology.
  - Implemented single-cell RNA sequencing exploratory data analysis pipelines for stem-cell and cancer model systems.
  - Proposed selective inference methods to study tissue-specific expression quantitative trait loci.

- *Department of Computer Science, Stanford University | Sep 2014 – Sep 2016*  
  - Designed algorithms in C and C++ to improve both the speed and accuracy of current state-of-the-art sequence aligners when aligning linked-read data.

- *Wireless Communication Group, Olin College | Sep 2010 – May 2014*  
  - Proposed stochastic geometric interference models to extend the range of applications in Space-Division Multiple Access (SDMA) networks.

- *Graph Theory Research Group, Olin College | Dec 2011 – May 2014*  
  - Investigated frequency assignment in wireless networks through graph coloring.
  - Found the minimum span of the L(2,1)-labeling for representative families of graphs, including amalgamation graphs and edge-path replacement graphs.

- *Signal Processing and Communication Laboratory, HKUST | Jul 2013 – Sep 2013*  
  - Studied the impact of unsynchronized interferers in multi-antenna OFDM systems.
  - Discovered closed-form solutions for the performance of typical linear receivers.


**Selected Publications:** 
- **Improving ML-based information retrieval software with user-driven functional testing and defect class analysis** - **FST**, 2022
- **Automatically Authoring Regression Tests for Machine-Learning-Based Systems** - **ICSE** 2021 
- *Progenitor identification and SARS-CoV-2 infection in human distal lung organoids* - **Nature**, 2020 
- *Exploratory gene ontology analysis with interactive visualization* - **Scientific Reports**, 2019
- *Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning* - **Nature Methods**, 2017

Google Scholar link: https://scholar.google.com/citations?user=2EasRdEAAAAJ&hl=en
